commands/webhook.go

```diff
@@ -146,13 +203,26 @@ func createIncomingWebhookCmdF(c client.Client, command *cobra.Command, args []s
 	channelArg, _ := command.Flags().GetString("channel")
 	channel := getChannelFromChannelArg(c, channelArg)
 	if channel == nil {
-		return errors.New("Unable to find channel '" + channelArg + "'")
+		return errors.Errorf("Unable to find channel '%s'", channelArg)
 	}
 
 	userArg, _ := command.Flags().GetString("user")
 	user := getUserFromUserArg(c, userArg)
 	if user == nil {
-		return errors.New("Unable to find user '" + userArg + "'")
+		return errors.Errorf("Unable to find user '%s'", userArg)
+	}
+
+	var owner *model.User
+	ownerArg, _ := command.Flags().GetString("owner")
+	if viper.GetBool("local") && ownerArg == "" {
+		return errors.New("owner flag should be set when this command is run in local mode")
```

Comment:
```suggestion
		return errors.New("owner should be specified to run this command in local mode")
```

same suggestion for other occurrences.

Reply:
Addressed in dfb3338